tangelo/linq/translator/translate_qiskit.py

```diff
@@ -103,6 +115,48 @@ def translate_qiskit(source_circuit):
             (GATE_QISKIT[gate.name])(target_circuit, gate.target[0], gate.target[0])
         else:
             raise ValueError(f"Gate '{gate.name}' not supported on backend qiskit")
+
+    return target_circuit
+
+
+def translate_c_from_qiskit(source_circuit) -> Circuit:
+    """Take in a qiskit.QuantumCircuit, return an equivalent Circuit
+
+    Args:
+        source_circuit (qiskit.QuantumCircuit): quantum circuit in the abstract qiskit.QuantumCircuit format.
```

Comment:
"abstract" is what i used to qualify our (Tangelo's) circuits, so I dont think this sentence should include it